### ISSUE TITLE
Fetching Hasura schema via introspection: Attempt 9 of 10, retrying...

### ISSUE THREAD

**Issue by staked-alekseyk** [input-output-hk/cardano-graphql#429]
### What i did

Launched cardano node, cardano-db-sync, hasura, graphql
It was syncing but after several days i found that there is not enough resources on node
so i decided to restart node. 
I stopped graphql, then hasura, then cardano-node and cardano-db-sync

after restarting - 
cardano node works well 
```
...
[cardano-:cardano.node.ChainDB:Notice:246] [2021-02-16 15:51:38.32 UTC] Chain extended, new tip: 08e2eb3818cafbb829742e28ae1b95ae376448cf25c6d5fd05404ace1d775177 at slot 21924407
[cardano-:cardano.node.ChainDB:Notice:246] [2021-02-16 15:51:46.19 UTC] Chain extended, new tip: a372dc14778148bd9d81d7848ff2bf19f9fb7882593cc950d1ca20ca72fa8deb at slot 21924415
[cardano-:cardano.node.ChainDB:Notice:246] [2021-02-16 15:51:56.56 UTC] Chain extended, new tip: a737640fc8969e6557b0d00f8b156cc08eda828b105ec5f268df6c97f3efc9a5 at slot 21924425
```
cardano-db-sync stuck on this
```
[db-sync-node.Mux:Info:68] [2021-02-16 14:59:25.44 UTC] Bearer on ConnectionId {localAddress = LocalAddress "", remoteAddress = LocalAddress "/home/cardano/ipc/ipc.socket"} event: Handshake Client end, duration 0.000442888s
[db-sync-node.Mux:Info:69] [2021-02-16 14:59:25.44 UTC] Bearer on ConnectionId {localAddress = LocalAddress "", remoteAddress = LocalAddress "/home/cardano/ipc/ipc.socket"} event: State: Mature
[db-sync-node:Info:72] [2021-02-16 14:59:25.44 UTC] Starting chainSyncClient
[db-sync-node.Subscription:Notice:32] [2021-02-16 14:59:25.47 UTC] [String "Required subscriptions started",String "SubscriptionTrace"]
[db-sync-node:Info:72] [2021-02-16 14:59:25.53 UTC] Database tip is at slot 21798036, block 5344260
[db-sync-node:Info:77] [2021-02-16 14:59:25.53 UTC] Running DB thread
[db-sync-node:Info:77] [2021-02-16 14:59:25.72 UTC] Rolling back to slot 21798036, hash 1713f882833c59dc063cda8c673a36128fb35eb5253386eba95c39eae2436091
[db-sync-node:Info:77] [2021-02-16 14:59:25.72 UTC] Deleting slots numbered: []
[db-sync-node:Info:80] [2021-02-16 14:59:26.83 UTC] getHistoryInterpreter: acquired
```
cardano-graphql has next errors
```
DbClient.pgSubscriber: Connected
Remote schema is missing Block
Fetching Hasura schema via introspection: Attempt 1 of 10, retrying...
Remote schema is missing Block
Fetching Hasura schema via introspection: Attempt 2 of 10, retrying...
{"level":"info","msg":"Help us improve Hasura! The cli collects anonymized usage stats which\nallow us to keep improving Hasura at warp speed. To opt-out or read more,\nvisit https://hasura.io/docs/1.0/graphql/manual/guides/telemetry.html\n","time":"2021-02-16T15:39:57Z"}
{"level":"info","msg":"Applying migrations...","time":"2021-02-16T15:39:57Z"}
{"level":"info","msg":"nothing to apply","time":"2021-02-16T15:39:57Z"}
Remote schema is missing Block
Fetching Hasura schema via introspection: Attempt 3 of 10, retrying...
Remote schema is missing Block
Fetching Hasura schema via introspection: Attempt 4 of 10, retrying...
Remote schema is missing Block
Fetching Hasura schema via introspection: Attempt 5 of 10, retrying...
Remote schema is missing Block
Fetching Hasura schema via introspection: Attempt 6 of 10, retrying...
Remote schema is missing Block
Fetching Hasura schema via introspection: Attempt 7 of 10, retrying...
Remote schema is missing Block
Fetching Hasura schema via introspection: Attempt 8 of 10, retrying...
Remote schema is missing Block
Fetching Hasura schema via introspection: Attempt 9 of 10, retrying...
Remote schema is missing Block
Fetching Hasura schema via introspection: Attempt 10 of 10, exiting
Remote schema is missing Block
```
hasura has  alot of messages
```
{"type":"http-log","timestamp":"2021-02-16T15:51:01.079+0000","level":"info","detail":{"operation":{"query_execution_time":1.9386e-5,"user_vars":{"x-hasura-role":"admin"},"request_id":"91e9b3c9-abf2-4407-9ca8-a072ac107845","response_size":157,"request_read_time":7.645e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1alpha1/config","ip":"100.96.149.52","method":"GET","content_encoding":"gzip"}}}
{"type":"http-log","timestamp":"2021-02-16T15:51:01.079+0000","level":"info","detail":{"operation":{"query_execution_time":2.2287e-3,"user_vars":{"x-hasura-role":"admin"},"request_id":"a253d3e4-8d5b-470c-a58e-2362d33ef3a4","response_size":91,"request_read_time":3.588e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1/query","ip":"100.96.149.52","method":"POST","content_encoding":"gzip"}}}
{"type":"http-log","timestamp":"2021-02-16T15:51:01.079+0000","level":"info","detail":{"operation":{"query_execution_time":1.1601e-5,"user_vars":{"x-hasura-role":"admin"},"request_id":"4f78e4fa-8b95-4dd1-a2d2-8cc13a31b711","response_size":157,"request_read_time":3.623e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1alpha1/config","ip":"100.96.149.52","method":"GET","content_encoding":"gzip"}}}
{"type":"http-log","timestamp":"2021-02-16T15:51:01.079+0000","level":"info","detail":{"operation":{"query_execution_time":3.180025e-3,"user_vars":{"x-hasura-role":"admin"},"request_id":"a2919d87-3859-46d2-bec3-93880b4e076b","response_size":68,"request_read_time":3.416e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1/query","ip":"100.96.149.52","method":"POST","content_encoding":"gzip"}}}
{"type":"http-log","timestamp":"2021-02-16T15:51:01.079+0000","level":"info","detail":{"operation":{"query_execution_time":2.1248253e-2,"user_vars":{"x-hasura-role":"admin"},"request_id":"e83ce6a4-4297-43aa-9c5a-1197a903eae4","response_size":70,"request_read_time":3.478e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1/query","ip":"100.96.149.52","method":"POST","content_encoding":"gzip"}}}
{"type":"http-log","timestamp":"2021-02-16T15:51:01.079+0000","level":"info","detail":{"operation":{"query_execution_time":2.708766e-3,"user_vars":{"x-hasura-role":"admin"},"request_id":"dfccede6-2a85-4b00-8470-d83520e27035","response_size":68,"request_read_time":3.286e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1/query","ip":"100.96.149.52","method":"POST","content_encoding":"gzip"}}}
{"type":"query-log","timestamp":"2021-02-16T15:51:04.903+0000","level":"info","detail":{"request_id":"85d0b8a4-0f16-4e50-88f6-01d0f9aef110","generated_sql":{"__schema":null},"query":{"query":"query IntrospectionQuery {\n  __schema {\n    queryType {\n      name\n    }\n    mutationType {\n      name\n    }\n    subscriptionType {\n      name\n    }\n    types {\n      ...FullType\n    }\n    directives {\n      name\n      description\n      locations\n      args {\n        ...InputValue\n      }\n    }\n  }\n}\n\nfragment FullType on __Type {\n  kind\n  name\n  description\n  fields(includeDeprecated: true) {\n    name\n    description\n    args {\n      ...InputValue\n    }\n    type {\n      ...TypeRef\n    }\n    isDeprecated\n    deprecationReason\n  }\n  inputFields {\n    ...InputValue\n  }\n  interfaces {\n    ...TypeRef\n  }\n  enumValues(includeDeprecated: true) {\n    name\n    description\n    isDeprecated\n    deprecationReason\n  }\n  possibleTypes {\n    ...TypeRef\n  }\n}\n\nfragment InputValue on __InputValue {\n  name\n  description\n  type {\n    ...TypeRef\n  }\n  defaultValue\n}\n\nfragment TypeRef on __Type {\n  kind\n  name\n  ofType {\n    kind\n    name\n    ofType {\n      kind\n      name\n      ofType {\n        kind\n        name\n        ofType {\n          kind\n          name\n          ofType {\n            kind\n            name\n            ofType {\n              kind\n              name\n              ofType {\n                kind\n                name\n              }\n            }\n          }\n        }\n      }\n    }\n  }\n}\n"}}}
{"type":"http-log","timestamp":"2021-02-16T15:51:04.903+0000","level":"info","detail":{"operation":{"query_execution_time":3.34157e-4,"user_vars":{"x-hasura-role":"cardano-graphql"},"request_id":"85d0b8a4-0f16-4e50-88f6-01d0f9aef110","response_size":4763,"request_read_time":6.205e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1/graphql","ip":"100.96.149.52","method":"POST","content_encoding":"gzip"}}}
{"type":"query-log","timestamp":"2021-02-16T15:51:10.296+0000","level":"info","detail":{"request_id":"1b5c21ef-6486-4710-90ed-9a77fd0f48ca","generated_sql":{"__schema":null},"query":{"query":"query IntrospectionQuery {\n  __schema {\n    queryType {\n      name\n    }\n    mutationType {\n      name\n    }\n    subscriptionType {\n      name\n    }\n    types {\n      ...FullType\n    }\n    directives {\n      name\n      description\n      locations\n      args {\n        ...InputValue\n      }\n    }\n  }\n}\n\nfragment FullType on __Type {\n  kind\n  name\n  description\n  fields(includeDeprecated: true) {\n    name\n    description\n    args {\n      ...InputValue\n    }\n    type {\n      ...TypeRef\n    }\n    isDeprecated\n    deprecationReason\n  }\n  inputFields {\n    ...InputValue\n  }\n  interfaces {\n    ...TypeRef\n  }\n  enumValues(includeDeprecated: true) {\n    name\n    description\n    isDeprecated\n    deprecationReason\n  }\n  possibleTypes {\n    ...TypeRef\n  }\n}\n\nfragment InputValue on __InputValue {\n  name\n  description\n  type {\n    ...TypeRef\n  }\n  defaultValue\n}\n\nfragment TypeRef on __Type {\n  kind\n  name\n  ofType {\n    kind\n    name\n    ofType {\n      kind\n      name\n      ofType {\n        kind\n        name\n        ofType {\n          kind\n          name\n          ofType {\n            kind\n            name\n            ofType {\n              kind\n              name\n              ofType {\n                kind\n                name\n              }\n            }\n          }\n        }\n      }\n    }\n  }\n}\n"}}}
{"type":"http-log","timestamp":"2021-02-16T15:51:10.296+0000","level":"info","detail":{"operation":{"query_execution_time":3.83684e-4,"user_vars":{"x-hasura-role":"cardano-graphql"},"request_id":"1b5c21ef-6486-4710-90ed-9a77fd0f48ca","response_size":4763,"request_read_time":6.565e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1/graphql","ip":"100.96.149.52","method":"POST","content_encoding":"gzip"}}}
{"type":"query-log","timestamp":"2021-02-16T15:51:19.730+0000","level":"info","detail":{"request_id":"13d7aeb4-069b-4fa7-9a17-5ac151a7283b","generated_sql":{"__schema":null},"query":{"query":"query IntrospectionQuery {\n  __schema {\n    queryType {\n      name\n    }\n    mutationType {\n      name\n    }\n    subscriptionType {\n      name\n    }\n    types {\n      ...FullType\n    }\n    directives {\n      name\n      description\n      locations\n      args {\n        ...InputValue\n      }\n    }\n  }\n}\n\nfragment FullType on __Type {\n  kind\n  name\n  description\n  fields(includeDeprecated: true) {\n    name\n    description\n    args {\n      ...InputValue\n    }\n    type {\n      ...TypeRef\n    }\n    isDeprecated\n    deprecationReason\n  }\n  inputFields {\n    ...InputValue\n  }\n  interfaces {\n    ...TypeRef\n  }\n  enumValues(includeDeprecated: true) {\n    name\n    description\n    isDeprecated\n    deprecationReason\n  }\n  possibleTypes {\n    ...TypeRef\n  }\n}\n\nfragment InputValue on __InputValue {\n  name\n  description\n  type {\n    ...TypeRef\n  }\n  defaultValue\n}\n\nfragment TypeRef on __Type {\n  kind\n  name\n  ofType {\n    kind\n    name\n    ofType {\n      kind\n      name\n      ofType {\n        kind\n        name\n        ofType {\n          kind\n          name\n          ofType {\n            kind\n            name\n            ofType {\n              kind\n              name\n              ofType {\n                kind\n                name\n              }\n            }\n          }\n        }\n      }\n    }\n  }\n}\n"}}}
{"type":"http-log","timestamp":"2021-02-16T15:51:19.730+0000","level":"info","detail":{"operation":{"query_execution_time":3.37556e-4,"user_vars":{"x-hasura-role":"cardano-graphql"},"request_id":"13d7aeb4-069b-4fa7-9a17-5ac151a7283b","response_size":4763,"request_read_time":6.478e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1/graphql","ip":"100.96.149.52","method":"POST","content_encoding":"gzip"}}}
{"type":"query-log","timestamp":"2021-02-16T15:51:36.181+0000","level":"info","detail":{"request_id":"c3eb489b-f2c6-4612-afe7-9a42822e1808","generated_sql":{"__schema":null},"query":{"query":"query IntrospectionQuery {\n  __schema {\n    queryType {\n      name\n    }\n    mutationType {\n      name\n    }\n    subscriptionType {\n      name\n    }\n    types {\n      ...FullType\n    }\n    directives {\n      name\n      description\n      locations\n      args {\n        ...InputValue\n      }\n    }\n  }\n}\n\nfragment FullType on __Type {\n  kind\n  name\n  description\n  fields(includeDeprecated: true) {\n    name\n    description\n    args {\n      ...InputValue\n    }\n    type {\n      ...TypeRef\n    }\n    isDeprecated\n    deprecationReason\n  }\n  inputFields {\n    ...InputValue\n  }\n  interfaces {\n    ...TypeRef\n  }\n  enumValues(includeDeprecated: true) {\n    name\n    description\n    isDeprecated\n    deprecationReason\n  }\n  possibleTypes {\n    ...TypeRef\n  }\n}\n\nfragment InputValue on __InputValue {\n  name\n  description\n  type {\n    ...TypeRef\n  }\n  defaultValue\n}\n\nfragment TypeRef on __Type {\n  kind\n  name\n  ofType {\n    kind\n    name\n    ofType {\n      kind\n      name\n      ofType {\n        kind\n        name\n        ofType {\n          kind\n          name\n          ofType {\n            kind\n            name\n            ofType {\n              kind\n              name\n              ofType {\n                kind\n                name\n              }\n            }\n          }\n        }\n      }\n    }\n  }\n}\n"}}}
{"type":"http-log","timestamp":"2021-02-16T15:51:36.181+0000","level":"info","detail":{"operation":{"query_execution_time":3.79204e-4,"user_vars":{"x-hasura-role":"cardano-graphql"},"request_id":"c3eb489b-f2c6-4612-afe7-9a42822e1808","response_size":4763,"request_read_time":5.791e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1/graphql","ip":"100.96.149.52","method":"POST","content_encoding":"gzip"}}}
{"type":"query-log","timestamp":"2021-02-16T15:52:04.943+0000","level":"info","detail":{"request_id":"9e984b4c-2cd0-4f58-92b6-278eca6452d9","generated_sql":{"__schema":null},"query":{"query":"query IntrospectionQuery {\n  __schema {\n    queryType {\n      name\n    }\n    mutationType {\n      name\n    }\n    subscriptionType {\n      name\n    }\n    types {\n      ...FullType\n    }\n    directives {\n      name\n      description\n      locations\n      args {\n        ...InputValue\n      }\n    }\n  }\n}\n\nfragment FullType on __Type {\n  kind\n  name\n  description\n  fields(includeDeprecated: true) {\n    name\n    description\n    args {\n      ...InputValue\n    }\n    type {\n      ...TypeRef\n    }\n    isDeprecated\n    deprecationReason\n  }\n  inputFields {\n    ...InputValue\n  }\n  interfaces {\n    ...TypeRef\n  }\n  enumValues(includeDeprecated: true) {\n    name\n    description\n    isDeprecated\n    deprecationReason\n  }\n  possibleTypes {\n    ...TypeRef\n  }\n}\n\nfragment InputValue on __InputValue {\n  name\n  description\n  type {\n    ...TypeRef\n  }\n  defaultValue\n}\n\nfragment TypeRef on __Type {\n  kind\n  name\n  ofType {\n    kind\n    name\n    ofType {\n      kind\n      name\n      ofType {\n        kind\n        name\n        ofType {\n          kind\n          name\n          ofType {\n            kind\n            name\n            ofType {\n              kind\n              name\n              ofType {\n                kind\n                name\n              }\n            }\n          }\n        }\n      }\n    }\n  }\n}\n"}}}
{"type":"http-log","timestamp":"2021-02-16T15:52:04.943+0000","level":"info","detail":{"operation":{"query_execution_time":4.44959e-4,"user_vars":{"x-hasura-role":"cardano-graphql"},"request_id":"9e984b4c-2cd0-4f58-92b6-278eca6452d9","response_size":4763,"request_read_time":6.242e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1/graphql","ip":"100.96.149.52","method":"POST","content_encoding":"gzip"}}}
{"type":"query-log","timestamp":"2021-02-16T15:52:55.269+0000","level":"info","detail":{"request_id":"74f9417f-18ca-4c01-9ee7-f218c16827ad","generated_sql":{"__schema":null},"query":{"query":"query IntrospectionQuery {\n  __schema {\n    queryType {\n      name\n    }\n    mutationType {\n      name\n    }\n    subscriptionType {\n      name\n    }\n    types {\n      ...FullType\n    }\n    directives {\n      name\n      description\n      locations\n      args {\n        ...InputValue\n      }\n    }\n  }\n}\n\nfragment FullType on __Type {\n  kind\n  name\n  description\n  fields(includeDeprecated: true) {\n    name\n    description\n    args {\n      ...InputValue\n    }\n    type {\n      ...TypeRef\n    }\n    isDeprecated\n    deprecationReason\n  }\n  inputFields {\n    ...InputValue\n  }\n  interfaces {\n    ...TypeRef\n  }\n  enumValues(includeDeprecated: true) {\n    name\n    description\n    isDeprecated\n    deprecationReason\n  }\n  possibleTypes {\n    ...TypeRef\n  }\n}\n\nfragment InputValue on __InputValue {\n  name\n  description\n  type {\n    ...TypeRef\n  }\n  defaultValue\n}\n\nfragment TypeRef on __Type {\n  kind\n  name\n  ofType {\n    kind\n    name\n    ofType {\n      kind\n      name\n      ofType {\n        kind\n        name\n        ofType {\n          kind\n          name\n          ofType {\n            kind\n            name\n            ofType {\n              kind\n              name\n              ofType {\n                kind\n                name\n              }\n            }\n          }\n        }\n      }\n    }\n  }\n}\n"}}}
{"type":"http-log","timestamp":"2021-02-16T15:52:55.269+0000","level":"info","detail":{"operation":{"query_execution_time":3.34538e-4,"user_vars":{"x-hasura-role":"cardano-graphql"},"request_id":"74f9417f-18ca-4c01-9ee7-f218c16827ad","response_size":4763,"request_read_time":5.234e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1/graphql","ip":"100.96.149.52","method":"POST","content_encoding":"gzip"}}}
{"type":"query-log","timestamp":"2021-02-16T15:54:23.274+0000","level":"info","detail":{"request_id":"637984b9-d604-4cd9-9ffb-3fe2d8aef15f","generated_sql":{"__schema":null},"query":{"query":"query IntrospectionQuery {\n  __schema {\n    queryType {\n      name\n    }\n    mutationType {\n      name\n    }\n    subscriptionType {\n      name\n    }\n    types {\n      ...FullType\n    }\n    directives {\n      name\n      description\n      locations\n      args {\n        ...InputValue\n      }\n    }\n  }\n}\n\nfragment FullType on __Type {\n  kind\n  name\n  description\n  fields(includeDeprecated: true) {\n    name\n    description\n    args {\n      ...InputValue\n    }\n    type {\n      ...TypeRef\n    }\n    isDeprecated\n    deprecationReason\n  }\n  inputFields {\n    ...InputValue\n  }\n  interfaces {\n    ...TypeRef\n  }\n  enumValues(includeDeprecated: true) {\n    name\n    description\n    isDeprecated\n    deprecationReason\n  }\n  possibleTypes {\n    ...TypeRef\n  }\n}\n\nfragment InputValue on __InputValue {\n  name\n  description\n  type {\n    ...TypeRef\n  }\n  defaultValue\n}\n\nfragment TypeRef on __Type {\n  kind\n  name\n  ofType {\n    kind\n    name\n    ofType {\n      kind\n      name\n      ofType {\n        kind\n        name\n        ofType {\n          kind\n          name\n          ofType {\n            kind\n            name\n            ofType {\n              kind\n              name\n              ofType {\n                kind\n                name\n              }\n            }\n          }\n        }\n      }\n    }\n  }\n}\n"}}}
{"type":"http-log","timestamp":"2021-02-16T15:54:23.274+0000","level":"info","detail":{"operation":{"query_execution_time":3.64189e-4,"user_vars":{"x-hasura-role":"cardano-graphql"},"request_id":"637984b9-d604-4cd9-9ffb-3fe2d8aef15f","response_size":4763,"request_read_time":6.363e-6},"http_info":{"status":200,"http_version":"HTTP/1.1","url":"/v1/graphql","ip":"100.96.149.52","method":"POST","content_encoding":"gzip"}
```

**Comment by staked-alekseyk** [comment]
any help?

**Comment by CyberCyclone** [comment]
How long did you leave it for? Typically it takes about 10 min for cardano-node to start up.

**Comment by rhyslbw** [comment]
Please upgrade to `3.2.0`. `cardano-db-sync@7.1.0` has some instabilities that have been addressed in `8.0.0`
